File: PYTHON/Problema1012.py
A, B, C =  map(float, input().split())

tri = (A*C)/2
cir = 3.14159*C**2
tra = ((A+B)*C)/2
qua = B**2
ret = A*B

print("TRIANGULO = %.3f" % (tri))
print("CIRCULO = %.3f" % (cir))
print("TRAPEZIO = %.3f" % (tra))
print("QUADRADO = %.3f" % (qua))
print("RETANGULO = %.3f" % (ret))

File: C/Problema1012.c
#import <stdio.h>

int main(){
	
	float A, B, C;
	scanf("%f %f %f", &A, &B, &C);

	float tri, cir, tra, qua, ret;
	tri = (A*C)/2;
	cir = 3.14159*C*C;
	tra = ((A+B)*C)/2;	
	qua = B*B;
	ret = A*B;

	printf("TRIANGULO: %.3f\n", tri);
	printf("CIRCULO: %.3f\n", cir);
	printf("TRAPEZIO: %.3f\n", tra);
	printf("QUADRADO: %.3f\n", qua);
	printf("RETANGULO: %.3f\n", ret);

}
